carp.common/src/commonMain/kotlin/dk/cachet/carp/common/application/data/CarpDataTypes.kt

```diff
@@ -53,10 +53,16 @@ object CarpDataTypes : DataTypeMetaDataMap()
 
     internal const val NON_GRAVITATIONAL_ACCELERATION_TYPE_NAME = "$CARP_NAMESPACE.nongravitationalacceleration"
     /**
-     * Acceleration along perpendicular x, y, and z axes, excluding gravity.
+     * Change in velocity along perpendicular x, y, and z axes in the device's coordinate system, excluding gravity.
```

Comment:
Rate of change of velocity over time = acceleration. Acceleration is still the accurate term here.